src/main/schemas/projectoraspectratios.schema.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$id": "http://isdcf.com/ns/json-schemas/registries/projectoraspectratios/1.0.0-beta.1",
+  "$comment": "Copyright, ISDCF <info@isdcf.com>",
+  "title": "Schema for the Projector Aspect Ratios and Resolutions Registry of the ISDCF",
+  "definitions": {
+    "dcncCode" : {
+        "type": "string",
+        "pattern": "^\\[?[A-Z0-9]\\]?"
+    },
+    "cplMetadata": {
+      "type": "object",
+      "properties": {
+        "definingDoc": {
+          "type": "array"
```

Comment:
```suggestion
              "type": "array",
              "items": {
                "type": "number"
              }
```

Reply:
These values aren't only numbers, the contain values such as:
- "SMPTE ST 428-12"

Reply:
I meant `"type": "string"`

Apologies.

Reply:
All good, updated in 23dea0a